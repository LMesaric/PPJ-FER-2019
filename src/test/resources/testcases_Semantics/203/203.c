int a = b = c = d = f = g = 1;
