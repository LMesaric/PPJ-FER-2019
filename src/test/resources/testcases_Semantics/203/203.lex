KR_INT 1 int
IDN 1 a
OP_PRIDRUZI 1 =
IDN 1 b
OP_PRIDRUZI 1 =
IDN 1 c
OP_PRIDRUZI 1 =
IDN 1 d
OP_PRIDRUZI 1 =
IDN 1 f
OP_PRIDRUZI 1 =
IDN 1 g
OP_PRIDRUZI 1 =
BROJ 1 1
TOCKAZAREZ 1 ;
