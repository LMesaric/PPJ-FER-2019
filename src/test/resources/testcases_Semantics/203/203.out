<primarni_izraz> ::= IDN(1,b)
